### DISCUSSION TITLE
关于bilibili组件的一些建议

### DISCUSSION THREAD

**Discussion by MonoLogueChi** [vuepress-theme-hope/vuepress-theme-hope#2269] [Q&A] [answered]
1. bilibili外部播放器现在已经没有高清选项了，即使下面显示720P，实际画面也是360P。
2. 现在使用的是固定高度的方式，不能很好的适配移动端，建议采用根据宽度计算高度的方式，可以参考[Bilibili.vue](https://github.com/u2sb/vuepress-plugin-smplayer/blob/v2/src/components/Bilibili/Bilibili.vue)

**Comment by Mister-Hope** [comment]
PR welcomed

**Comment by Mister-Hope** [comment, marked as answer]
Feature supported.
